.github/workflows/security_owasp.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         distribution: 'temurin'
     - name: Run gradle OWASP Dependency Check Analyse
       id: owasp-analyse
-      run: ./gradlew ${{ inputs.subproject }}:dependencyCheckAnalyze --info
+      run: ./gradlew ${{ inputs.subproject }}:dependencyCheckAnalyze -Pdatafeed.url='file:///opt/vulnz/cache' --info
```

Comment:
I did toy with putting this as a project property with a default, but don't see the point as `hmpps-github-actions-runner-security` isn't a property at present and it relies on that runner anyway